features/src/main/feature/camel-features.xml

```diff
@@ -1403,6 +1403,28 @@
         <bundle dependency='true'>mvn:io.netty/netty-codec-http/4.1.107.Final</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-olingo4/${project.version}</bundle>
     </feature>
+    <feature name='camel-openstack' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature>jackson</feature>
+        <feature version='${guava-version}'>guava</feature>
+        <!-- use wrap to override the guava version -->
+        <bundle dependency='true'>wrap:mvn:com.github.openstack4j.core/openstack4j-core/${openstack4j-version}$overwrite=merge&amp;Import-Package=com.fasterxml*,com.github*,javax*,org*,com.google.common.base;version=33.0.0</bundle>
```

Comment:
❓ instead of `com.google.common.base`, we can go with `com.google.common*`

Reply:
yes